packages/desktop-client/src/components/CommandBar.tsx

```diff
@@ -269,7 +269,7 @@ export function CommandBar() {
               color: 'var(--color-pageTextSubdued)',
             })}
           >
-            {t('No results found')}
```

Comment:
Do you know if we can lint against this pattern somehow?

Reply:
I'm honestly not sure, it would be brilliant if we could - especially if we could get autofix working. The problem is it's quite hard even to grep for, let alone write a rule around.

Reply:
True, it would get pretty complex. We can maybe set up more general i18n linters in the future